pkg/asset/quota/quota.go

```diff
@@ -118,7 +122,23 @@ func (a *PlatformQuotaCheck) Generate(dependencies asset.Parents) error {
 			return summarizeFailingReport(reports)
 		}
 		summarizeReport(reports)
-	case azure.Name, baremetal.Name, libvirt.Name, none.Name, openstack.Name, ovirt.Name, vsphere.Name:
+	case typesopenstack.Name:
+		opts := &openstackclientconfig.ClientOpts{Cloud: ic.Config.Platform.OpenStack.Cloud}
+		os.Unsetenv("OS_CLOUD")
```

Comment:
I really wish we could wrap that unset OS_CLOUD somehow.

Reply:
Yeah I know, this was mentioned here: https://github.com/openshift/installer/pull/4426#issuecomment-734891653

I propose that we leave it here for now, and submit a separated PR to address this valid issue.

Reply:
nevermind, I managed to get rid of it when using CloudInfo struct. Thanks!